core/trino-main/src/main/resources/webapp/references.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+
+    <meta name="description" content="References - Trino">
+    <title>References - Trino</title>
+
+    <link rel="icon" href="assets/favicon.ico">
+
+    <!-- Bootstrap core -->
+    <link href="vendor/bootstrap/css/bootstrap.css" rel="stylesheet">
+
+    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
+    <!--[if lt IE 9]>
```

Comment:
What is IE8?

Reply:
that's what i thought. But why would we want to carry IE8 support in new code?
its last release was in 2011

Reply:
This comment and the scripts mentioned by it are also present in other components of Trino's Web UI, such as:
- [disabled.html](https://github.com/trinodb/trino/blob/master/core/trino-main/src/main/resources/webapp/disabled.html#L28-L32)
- [embedded_plan.html](https://github.com/trinodb/trino/blob/master/core/trino-main/src/main/resources/webapp/embedded_plan.html#L15-L19)
- [index.html](https://github.com/trinodb/trino/blob/master/core/trino-main/src/main/resources/webapp/index.html#L15-L19)
- [login.html](https://github.com/trinodb/trino/blob/master/core/trino-main/src/main/resources/webapp/login.html#L28-L32)
- [plan.html](https://github.com/trinodb/trino/blob/master/core/trino-main/src/main/resources/webapp/plan.html#L15-L19)
- [query.html](https://github.com/trinodb/trino/blob/master/core/trino-main/src/main/resources/webapp/query.html#L15-L19)
- [stage.html](https://github.com/trinodb/trino/blob/master/core/trino-main/src/main/resources/webapp/stage.html#L15-L19)
- [timeline.html](https://github.com/trinodb/trino/blob/master/core/trino-main/src/main/resources/webapp/timeline.html#L19-L23)
- [worker.html](https://github.com/trinodb/trino/blob/master/core/trino-main/src/main/resources/webapp/worker.html#L15-L19)
- [workers.html](https://github.com/trinodb/trino/blob/master/core/trino-main/src/main/resources/webapp/workers.html#L15-L19)
- [oauth2/logout.html](https://github.com/trinodb/trino/blob/master/core/trino-main/src/main/resources/oauth2/logout.html#L28-L32)
- [oauth2/success.html](https://github.com/trinodb/trino/blob/master/core/trino-main/src/main/resources/oauth2/success.html#L28-L32)

My assumption based on this is that we should keep the compatibility (hence the addition of the scripts on my changeset) or clean it up across the board. Which one makes more sense?

Reply:
there is no benefit of copying dead code. It's OK (and preferrable) to cleanup the existing files in a separate PR